pandas/tests/tools/test_to_datetime.py

```diff
@@ -671,17 +675,47 @@ def test_to_datetime_mixed_datetime_and_string_with_format(
         tm.assert_index_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "fmt, utc, expected",
+        "fmt",
+        ["%Y-%d-%m %H:%M:%S%z", "%Y-%m-%d %H:%M:%S%z"],
+        ids=["non-ISO8601 format", "ISO8601 format"],
+    )
+    @pytest.mark.parametrize(
+        "utc, args, expected",
```

Comment:
is there a single `pytest.param` in this parametrisation here now? if so, we can just remove it and inline it in the test

Reply:
Thanks, you are right, we have a single `pytest.param` in this parametrisation, so I removed it.